.github/workflows/test_on_push.yml

```diff
@@ -38,11 +38,6 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-12, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        # We check coverage on Ubuntu with Python 3.11, so we skip unit tests for it here
-        # TODO: check coverage with Python 3.12 when [odes] supports it
-        exclude:
-          - os: ubuntu-latest
-            python-version: "3.11"
```

Comment:
i.e., exclude Ubuntu and Python 3.12 here (one cannot suggest changes on removed lines)

Reply:
See above